src/hotspot/share/runtime/abstract_vm_version.hpp

```diff
@@ -56,6 +56,9 @@ class Abstract_VM_Version: AllStatic {
 
   // CPU feature flags, can be affected by VM settings.
   static uint64_t _features;
+  // Extra CPU feature flags used when all 64 bits of _features are exhausted for
+  // on a given target, currently only used for x86_64, can be affected by VM settings.
+  static uint64_t _extra_features;
```

Comment:
That's unfortunate. Maybe it's time to turn `_features` into a fixed size (platform-specific) bitmap instead? (`RegMask` is one existing example.) Having 2 independent fields is error-prone (look at  `_cpu_features`).